Assets/Scripts/UI/Handlers/LoadSceneButton.cs

```diff
@@ -1,10 +1,20 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-public class LoadSceneButton : MonoBehaviour {
-    [SerializeField] private string nameOfSceneToLoad;
+public class LoadSceneButton : MonoBehaviour
+{
+    [SerializeField]
+    private string nameOfSceneToLoad;
 
-    public void LoadSceneButton_OnClick() {
-        SceneManager.LoadScene(nameOfSceneToLoad);
+    public void LoadSceneButton_OnClick()
+    {
+        try
+        {
+            SceneManager.LoadScene(nameOfSceneToLoad);
+        }
+        catch
+        {
+            Debug.Log("Scene has not been implemented yet.");

```

Comment:
I think it's better to use `$"Scene {nameOfSceneToLoad} not found."` or something like that so that more information is provided about the scene name that failed to load. Having the scene name there would be helpful in determining if `nameOfSceneToLoad` is wrong.